src/formats/subrip/subripinputformat.h

```diff
@@ -48,7 +48,7 @@ class SubRipInputFormat : public InputFormat
 
 			offset += m_regExp.matchedLength();
 
-			QString text(data.mid(offset, (unsigned)m_regExp.indexIn(data, offset) - offset));
+			QString text(data.mid(offset, m_regExp.indexIn(data, offset) - offset));
```

Comment:
No need to. Have tried it and works fine, if we ever see last line on .srt files loads empty or funny, we'll know qt changed how QString.mid() works :)